core/misc/collapse.js

```diff
@@ -69,8 +69,8 @@ Backdrop.behaviors.collapse = {
 
       var summary = $('<span class="summary"></span>');
       $fieldset.
-        bind('summaryUpdated', function () {
-          var text = $.trim($fieldset.backdropGetSummary());
+        on('summaryUpdated', function () {
+          var text = $fieldset.backdropGetSummary().trim();
```

Comment:
This is probably beyond the scope of this PR, but the custom jQueyr method `backdropGetSummary` defined in `form.js` already trims the return.

Reply:
Yep, I know. I didn't dare to skip the second run, but you give me more confidence that it won't cause too much discussion. :relaxed:

Reply:
I removed the redundant trim().